core/src/main/java/org/apache/gravitino/storage/relational/service/MetalakeMetaService.java

```diff
@@ -191,6 +192,10 @@ public boolean deleteMetalake(NameIdentifier ident, boolean cascade) {
                 SessionUtils.doWithoutCommit(
                     TableMetaMapper.class,
                     mapper -> mapper.softDeleteTableMetasByMetalakeId(metalakeId)),
+            () ->
+                SessionUtils.doWithoutCommit(
```

Comment:
I think it's also need add the recursive delete logic in CatalogMetaService, SchemaMetaService, right?

Reply:
I see, I will add them.